models/image.model.js

```diff
@@ -0,0 +1,25 @@
+const Sequelize = require('sequelize')
+const Connection = require('./connection')
+const Room = require('../models/room.model')
+
+const connection = new Connection()
+const images = connection.sequelize.define(
+  'images',
```

Comment:
@Bymatt10 Utiliza mayúsculas
```suggestion
  'Images',
```